mmselfsup/core/hooks/byol_hook.py

```diff
@@ -7,7 +7,7 @@
 
 @HOOKS.register_module()
 class BYOLHook(Hook):
```

Comment:
Can this hook be changed to a more general form with the name like EMAHook, and explain that this hook can be used in algorithms like BYOL, MoCo v3 in docstring.

Reply:
If we change name directly, it will have BC-breaking, the old configs downloaded by user are not compatible. It seems that the register_module() support multiple names, I will check that.